src/components/LifeEventPicker.tsx

```diff
@@ -1,54 +1,255 @@
+
+  function toggleMarriageTextField(): void {
+    setMarriage(!marriage);
+  }
+
+  function toggleDeathTextField(): void {
+    setDeath(!death);
+  }
+
+  function toggleAnyEventTextField(): void {
+    setAnyEvent(!anyEvent);
+  }
+
   return (
     <div className={classes.paper}>
       <Typography component="h1" variant="h6">
         Search with a life event
       </Typography>
-      <Typography style={{ marginBottom: 20, color: 'blue' }}>
-        Birth | Residence | Marriage | Death | Any
+      <Typography style={{ marginBottom: 20 }}>
+        <Link onClick={toggleBirthTextField}>Birth</Link>
+        {' | '}
+        <Link onClick={toggleResidenceTextField}>Residence</Link>
+        {' | '}
+        <Link onClick={toggleMarriageTextField}>Marriage</Link>
+        {' | '}
+        <Link onClick={toggleDeathTextField}>Death</Link>
+        {' | '}
+        <Link onClick={toggleAnyEventTextField}>Any</Link>
       </Typography>
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={6}>
-          <TextField
-            autoComplete="fname"
-            name="birthplace"
-            variant="outlined"
-            required
-            fullWidth
-            id="birthplace"
-            label="Birthplace"
-            autoFocus
-          />
-        </Grid>
-        <Grid item xs={6} sm={3}>
-          <TextField
-            variant="outlined"
-            required
-            fullWidth
-            id="startYear"
-            label="Year Start"
-            name="startYear"
-          />
-        </Grid>
-        <Grid item xs={6} sm={3}>
-          <TextField
-            variant="outlined"
-            required
-            fullWidth
-            id="endYear"
-            label="Year End"
-            name="endYear"
-          />
-        </Grid>
-      </Grid>
+
+      <div>
+        {birth ? (
+          <Grid container spacing={2} style={{ marginBottom: 20 }}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                name="birthplace"
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplace"
+                label="Birthplace"
+                autoFocus
+              />
+            </Grid>
+            <Grid item xs={6} sm={3}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplaceStartYear"
+                label="Year Start"
+                name="birthplaceStartYear"
+              />
+            </Grid>
+            <Grid item xs={6} sm={3}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="birthplaceEndYear"
+                label="Year End"
+                name="birthplaceEndYear"
+              />
+            </Grid>
+          </Grid>
+        ) : (
+          ''
+        )}
+      </div>
+      {/* ------------------- */}
+      <div>
```

Comment:
Maybe you just haven't committed yet, but I still see these <div> tags.

Reply:
Sorry, was updated and now pushed to Github!